### PR TITLE
Fix page load bug

### PR DESCRIPTION
**Pull request by bsstoner**
**Reviewer:**
@jdorweiler 

## Steps to test this PR:
Open new tab
start typing 'goo'
icon should no longer update to D
navigate to duckduckgo.com (without going to google first)
icon should update to A for duckduckgo

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: browsers/duckduckgo.safariextension/js/content-script.js [hunk 1]
```diff
@@ -51,14 +51,5 @@ function getLocation () {
 }
 
 window.onbeforeunload = ((e) => unload(e))
-window.onfocus = ((e) => sendLoadEvent(e))
```
Comment (bsstoner) [inline browsers/duckduckgo.safariextension/js/content-script.js]: what was the onfocus handler for? That may be broken now if you needed that code to run on focus.

**Comment by jdorweiler** [comment]
It's almost there. The badge doesn't update with the preloaded site grade but I still see wrong site data in the popup.

**Comment by bsstoner** [comment]
Pushed the changes we discussed, it solves the first bug, but creates some weird scenarios. e.g.
1) navigate to duckduckgo.com
2) start typing google.com, but don't hit enter and then click out/press escape
3) open the popup (still has info for google showing)

It seems like there are 2 tabs (the visible and the one preloading), maybe we need to model it that way? Leave the visible tab alone, but route the preloading events to a preloadingTab that we keep separate until it becomes visible, then we upgrade it to the real tab?

FF/Chrome didn't have this issue?

**Comment by laurengarcia** [comment]
This branch appears to fix the bug I was seeing earlier where private browsing tabs don't show the correct https state for sites like google.com and microsoft.com. I'll test it some more to make sure it hasn't broken other things.

**Comment by jdorweiler** [comment]
Looks good, thanks for fixing this.

**Pull request merged**